subprojects/gradle/lint-report/src/main/kotlin/com/avito/android/lint/slack/LintSlackReporter.kt

```diff
@@ -24,44 +26,113 @@ interface LintSlackReporter {
 
         override fun report(
             lintReport: LintReportModel,
-            channel: SlackChannel,
+            channelForDevs: SlackChannel,
+            channelForLintBugs: SlackChannel,
             buildUrl: HttpUrl
         ) {
-            if (shouldSendAlert(lintReport)) {
-                logger.debug("$tag: Sending lint alert...")
-
-                slackClient.uploadHtml(
-                    channel = channel,
-                    message = buildSlackMessage(lintReport, buildUrl),
-                    file = lintReport.htmlFile
-                ).fold(
-                    { logger.debug("$tag: Report sent successfully") },
-                    { error -> logger.critical("$tag: Can't send report ${error.message}") }
-                )
-            } else {
-                logger.debug("$tag Skip sending lint alert")
+            when (lintReport) {
+                is LintReportModel.Valid -> {
+                    val (errors, everythingElse) = lintReport.issues.partition { it.severity == LintIssue.Severity.ERROR }
+                    val (warnings, unknowns) = everythingElse.partition { it.severity == LintIssue.Severity.WARNING }
+                    val (fatalErrors, regularErrors) = errors.partition { it.isFatal }
+
+                    var isMessageSent = false
+
+                    if (regularErrors.isNotEmpty()) {
+                        sendReport(
+                            channel = channelForDevs,
+                            message = buildSlackMessageToDevs(
+                                projectPath = lintReport.projectRelativePath,
+                                errors = regularErrors,
```

Comment:
In this case, we hide some of the errors. 
I'm afraid it can be misleading to see different information in a message and in a full report.
I propose to add information about fatals but mark them.

Reply:
probably, let's see how it's going, it will be easy to fill another type of errors in separate line here